tests/unit/aiplatform/test_models.py

```diff
@@ -255,6 +255,8 @@ def get_model_with_custom_project_mock():
         get_model_mock.return_value = gca_model.Model(
             display_name=_TEST_MODEL_NAME,
             name=_TEST_MODEL_RESOURCE_NAME_CUSTOM_PROJECT,
+            artifact_uri=_TEST_ARTIFACT_URI,
+            description=_TEST_DESCRIPTION,
```

Comment:
These fields were added to test properties would return without throwing a `RuntimeError` if `gca_resource` is set, though these tests could have asserted for empty strings per underlying proto implementation so this may not be necessary.